Android/app/src/main/java/io/github/project_travel_mate/travel/Hotels.java

```diff
@@ -131,19 +132,45 @@ public void onDateSet(DatePickerDialog datePickerDialog, int year, int month, in
         mDate = day + "-";
         String monthString;
         switch (month + 1) {
-            case 1: monthString = "January";    break;
-            case 2: monthString = "February";   break;
-            case 3: monthString = "March";      break;
-            case 4: monthString = "April";      break;
-            case 5: monthString = "May";        break;
-            case 6: monthString = "June";       break;
-            case 7: monthString = "July";       break;
-            case 8: monthString = "August";     break;
-            case 9: monthString = "September";  break;
-            case 10:monthString = "October";    break;
-            case 11:monthString = "November";   break;
-            case 12:monthString = "December";   break;
-            default:monthString = "Invalid month"; break;
+            case 1:
```

Comment:
I think you can use this [stackoverflow answer](https://stackoverflow.com/q/5799140/5602759)